roles/confluent.ksql/tasks/set_principal.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Set Principal - Sasl Scram
+  set_fact:
+    ksql_log4j_principal: "{{ sasl_scram_users.admin.principal }}"
```

Comment:
Should ksql be acting as the kafka admin or the scram user set up for ksql?
sasl_scram_users.admin.principal or sasl_scram_users.ksql.principal